crash when page reload

```
[]
2021/10/23 23:44:56 http: response.WriteHeader on hijacked connection from ueckoken/plarail2021-soft-external/internal.unpackClientSendData (clientHandler.go:63)
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x8255ae]

goroutine 9 [running]:
ueckoken/plarail2021-soft-external/internal.clientHandler.ServeHTTP.func2()
	/mnt/live/projects/koken/plarail2021-soft/backend/external/internal/clientHandler.go:42 +0x2e
created by ueckoken/plarail2021-soft-external/internal.clientHandler.ServeHTTP
	/mnt/live/projects/koken/plarail2021-soft/backend/external/internal/clientHandler.go:40 +0x265
exit status 2
```

--- a/backend/external/internal/clientHandler.go
+++ b/backend/external/internal/clientHandler.go
@@ -37,7 +37,7 @@ func (m clientHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	var cChannel = clientChannel{cSync, cDone}
 	m.ClientChannelSend <- cChannel
 	go func() {
-		r, err := unpackClientSendData(w, c)
+		r, err := unpackClientSendData(c)
 		if err != nil {
 			log.Println(err)
 			return
@@ -59,7 +59,7 @@ func (m clientHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-func unpackClientSendData(w http.ResponseWriter, c *websocket.Conn) (*StationState, error) {
+func unpackClientSendData(c *websocket.Conn) (*StationState, error) {
 	_, msg, err := c.ReadMessage()
 	if err != nil {
 		return nil, fmt.Errorf("websocket read failed: %e", err)

